contracts/src/BeefyClient.sol

```diff
@@ -428,6 +443,27 @@ contract BeefyClient {
         return Bitfield.subsample(ticket.prevRandao, bitfield, ticket.numRequiredSignatures, ticket.validatorSetLen);
     }
 
+    function setInitialCheckpoint(
```

Comment:
Maybe we should call this `forceCheckpoint` to match the equivalent in the bacon client.

Reply:
Good idea! 🥓